target.py

```diff
@@ -20,21 +20,18 @@ class Target(pygame.sprite.Sprite):
     
     def __init__(self):
```

Comment:
Looks like we're combining your work with the target attributes to the target class. Looks good.

Reply:
Oh, do you mean in the update function's if statement?  The Rect object has x, y, width, and height attributes so I just used those.  Its width and height are from the target attributes when we scaled the image.